copulas/bivariate/__init__.py

```diff
@@ -156,7 +156,7 @@ def select_copula(X):
         np.concatenate((left, right)) for left, right in zip(
             candidate_left_auts, candidate_right_auts
         )
-    ]
+    ]  # noqa: JS102
```

Comment:
I don't know why it complains about this one...

Reply:
That's strange. Would it help to add commas? e.g.

```
candidate_auts = [
    np.concatenate((left, right)) for left, right in zip(
        candidate_left_auts, candidate_right_auts,
    ),
]
```

Reply:
I think this is because once you add any multi-line in an expression, the plugin expects everything to be multi-line, and also the fact that it is a comprehension messes it a bit.

I suggest to change the splitting to right before the `for`, and then the plugin will not complain:

```python3
    candidate_auts = [
        np.concatenate((left, right))
        for left, right in zip(candidate_left_auts, candidate_right_auts)
    ]
```

Reply:
@katxiao adding the comma doesn't work, I also thought of that 😀

@csala that works, thanks!